Add --ssh option and ssh verb (#594)
systemd/mkosi: mkosi.md

  built with `--bootable` (see below). This command must be executed
  as `root` unless the image already exists and `-f` is not specified.

`ssh`

: When the image is built with the `--ssh` option, this command connects

>   to a booted (`boot`, `qemu` verbs) container/vm via SSH. Make sure to

Reviewer: ```suggestion
  to a booted (`boot`, `qemu` verbs) container/VM via SSH. Make sure to
```

  run `mkosi ssh` with the same config as `mkosi build` was run with so
  that it has the necessary information available to connect to the running

>   container/vm via SSH.

Reviewer: ```suggestion
  container/VM via SSH.
```

  on file systems that support subvolume snapshots or 'reflinks' natively ("btrfs"
  or new "xfs") than on more traditional file systems that do not ("ext4").

`--ssh`

> : If specified, installs and enables sshd in the final image. Also generates
>   a public/private SSH keypair and adds the public key to the SSH authorized keys

Reviewer: ```suggestion
: If specified, installs and enables sshd in the final image and generates
  a SSH keypair and adds the public key to root's authorized keys
```

  in the final image. The private key is stored in mkosi's output directory. When
  building with this option and running the image using `mkosi boot` or `mkosi qemu`,
  the `mkosi ssh` command can be used to connect to the container/vm via SSH.

`--ssh-port`

> : Sets the port from the host that is forwarded to port 22 in the container/vm when

Reviewer: ```suggestion
: Sets the port from the host that is forwarded to port 22 in the container/VM when
```
